### PR TITLE
Add encrypted_wallets migration

### PR DESCRIPTION
**Pull request by Sajma27**
# Summary
A description of the changes proposed in the pull request.

Fixes #number

## Checklist
- [ ] Change is small and easy to review (please split big changes into multiple PRs)
- [ ] Change is consistent with architecture
- [ ] Change is consistent with test architecture
- [ ] Change is consistent with naming conventions
- [ ] New code is covered with tests
- [ ] Tests related to old code are updated
- [ ] Documentation is up to date with changes

### CODE REVIEW

File: universal-login-relayer/src/integration/sql/migrations/20200708000001_create_encrypted_wallets.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+/*
+  THIS IS A MIGRATION
+  This file SHOULD NOT be RENAMED or CHANGED in ANY WAY!
+  If you need to change the database schema create a new migration.
+*/
+
+exports.up = async (knex) => {
+  await knex.schema.createTable('encrypted_wallets', (table) => {
+    table.increments();
+    table.string('walletJSON').notNullable();
```
Comment (JustynaBroniszewska) [inline universal-login-relayer/src/integration/sql/migrations/20200708000001_create_encrypted_wallets.js:10]: What is the length limit of string in PSQL and what is the length of stringified walletJSON?

Comment (Sajma27) [reply universal-login-relayer/src/integration/sql/migrations/20200708000001_create_encrypted_wallets.js:10]: I think we can say that maximum walletJSON length would be 400. So this is way below length string limit of PSQL which is over 10000000

File: universal-login-relayer/src/integration/sql/migrations/20200708000001_create_encrypted_wallets.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  THIS IS A MIGRATION
+  This file SHOULD NOT be RENAMED or CHANGED in ANY WAY!
+  If you need to change the database schema create a new migration.
+*/
+
+exports.up = async (knex) => {
+  await knex.schema.createTable('encrypted_wallets', (table) => {
+    table.increments();
+    table.string('walletJSON', 900).notNullable();
+    table.string('email').notNullable();
```
Comment (xitronix) [inline universal-login-relayer/src/integration/sql/migrations/20200708000001_create_encrypted_wallets.js:11]: In my opinion, email should be the primary key or at least unique value.

Comment (Sajma27) [reply universal-login-relayer/src/integration/sql/migrations/20200708000001_create_encrypted_wallets.js]: Done

**Pull request merged**